### ISSUE TITLE
cannot open www.joomla.com

### ISSUE THREAD

**Issue by djtsov** [annulen/webkit#567]
QtTestBrowser return error: SSL handshake failed.
1) Tryed to ignore by usage the signal networkAccessManager::sslErrors, but it doesn't occur.
2) Tryed to use the reply.ignoreSslErrors(), witout result
3) Tryed to load the GlobalSign sertificate from disk and add it to QSslConfiguration, without result too.

**Comment by annulen** [comment]
Works here.

Please check you system clock and update system CA certificates

**Comment by djtsov** [comment]
Which version of OpenSSL are you using?

**Comment by annulen** [comment]
1.0.2 should work. Actually, if openssl version was incompatible, you wouldn't be able to open any https site

**Comment by djtsov** [comment]
Maybe deal in environment? Have I to set any environment variables for OpenSSL right work? I just append the /path/to/openssl/bin/ into PATH variable

**Comment by annulen** [comment]
No.

Have you checked if your system clock is right?

**Comment by djtsov** [comment]
Yes, I've checked a system clock. Now my local time 28.06.2017 03:30
